app/src/main/res/layout/la_time_measurement_card.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/ll_chart_channel"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/ll_margin_top"
+    android:background="@drawable/la_chart_custom_border"
+    android:orientation="vertical"
+    android:paddingBottom="@dimen/ll_padding_bottom"
+    android:paddingEnd="@dimen/ll_padding_right"
+    android:paddingLeft="@dimen/ll_padding_left"
+    android:paddingRight="@dimen/ll_padding_right"
+    android:paddingStart="@dimen/ll_padding_left"
+    android:paddingTop="@dimen/ll_padding_top">
+
+
+    <Spinner
+        android:id="@+id/channel_select_spinner_1"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:backgroundTint="@color/white"
+        android:gravity="center_horizontal"
+        android:popupBackground="@color/colorPrimary" />
+
+
```

Comment:
Why are we leaving two lines? Leaving one line is fine, so give all the files code reformat :+1: